entrypoint.sh

```diff
@@ -82,21 +98,24 @@ restore() {
         fi
     fi
 
-    debug "Decrypting $gpg_file"
-    gpg decrypt "$gpg_file"
-
-    debug "Moving ${gpg_file%%.gpg} to ${plain_file_path}"
-    mkdir -p "${SOURCE_PATH}" || bail "Couldn't create plain file destination directory (${SOURCE_PATH})"
-    mv "${gpg_file%%.gpg}" "${plain_file_path}"
+    debug "Decrypting '$gpg_file'"
+    gpg decrypt "$gpg_file" "${plain_file_path}"
 }
 
 list-backups() {
+    # source-path must exist
+    [ -e "${SOURCE_PATH}" ] ||
```

Comment:
Should this (and line 110) check for the path being a readable/executable directory?